### DISCUSSION TITLE
Filament Database Notification & broadcasting+ Laravel-websocket

### DISCUSSION THREAD

**Discussion by MarJose123** [filamentphp/filament#4737] [Help]
Hi, currently I enabled the  `database_notification` and `broadcasting` in my filament and upon checking my browser network the wss pusher request return this error message:

```
Connection Closed: 4001 App key LKJASLD not in this cluster. Did you forget to specify the cluster?
```

**Comment by danharrin** [comment]
Did you forget to change the `PUSHER_APP_CLUSTER`? I think that error is pretty self-explanatory

**Reply by MarJose123** [reply]
> Did you forget to change the `PUSHER_APP_CLUSTER`? I think that error is pretty self-explanatory

Nope, there is a value inside my `.env` file, `laravel-websocket` is working fine but on the filament-admin it showing that error.

Tried also in postman and the websocket is working correctly.

**Reply by danharrin** [reply]
Where have you told Filament to use that package instead of Pusher.com ? I've never used it so I don't know

**Reply by MarJose123** [reply]
I'm using  [larave-websocket](https://github.com/beyondcode/laravel-websockets) an it use the pusher.
